Error after installed web pack

I've installed[ web pack](http://docs.nativescript.org/angular/tooling/bundling-with-webpack.html) on my project and then i'm getting the following error.

**ERROR in ./modules/pack.address/pack.address.component.ts
Module not found: Error: Can't resolve 'nativescript-directions' in 'C:\dev\fm-transportes\src\FM.Transportes\app\modules\pack.address'
 @ ./modules/pack.address/pack.address.component.ts 13:17-51
 @ ./app.module.ts
 @ ./main.ts**

**database.service.ts**
```
const Sqlite = require("nativescript-sqlite");

const dbName = "fm.db";
import { Package, Pack, Order, Address, Reason, PackageSummary, PackStatus, Checkout } from "../";

export class DatabaseService {
  constructor() {
    this.Setup();    
  }
```

Is the nativescript-directions plugin compatible ?
I also opened an issue on the web pack page 
#https://github.com/NativeScript/nativescript-dev-webpack/issues/81

sorry, i past wrong code. I'm with error with 
sqlite still as you can see below:

```
ERROR in ./modules/shared/services/database.service.ts
Module not found: Error: Can't resolve 'nativescript-sqlite' in 'C:\dev\fm-transportes\src\FM.Transportes\app\modules\shared\services'
 @ ./modules/shared/services/database.service.ts 1:13-43
 @ ./modules/shared/index.ts
 @ ./app.module.ts
 @ ./main.ts
```

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "nativescript-sqlite",
   "version": "1.1.2",
   "description": "A sqlite NativeScript module for Android and iOS",
-  "main": "sqlite.js",
+  "main": "sqlite",
   "nativescript": {
     "platforms": {
       "android": "1.4.0",

